lib/msal-core/azure-pipelines.yml

```diff
@@ -27,3 +27,8 @@ jobs:
           archiveType: 'zip'
           archiveFile: '$(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip'
           replaceExistingArchive: true
+      - task: PublishBuildArtifacts@1
+        inputs:
+          PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+          ArtifactName: 'drop'
+          publishLocation: 'Container'
```

Comment:
@pkanher617 I accidently pulled master into my branch instead of dev, any issues with these changes going into dev?